examples/nlp/biobert_notebooks/biobert_qa.ipynb

```diff
@@ -178,6 +188,7 @@
     "qa_head.restore_from(head_checkpoint)\n",
     "task_loss = nemo_nlp.nm.losses.SpanningLoss()\n",
     "# create training data layer, preprocessing takes a while. If you want to cache preprocessed data for future reuse use --use_cache=True\n",
+    "# remember to delete the cache when you switch the tokenizer/model (BioBERT and BioMegatron use different tokenizers)\n",
```

Comment:
why? cased and uncased tokenizers have diff vocab sizes. are there any params missing to distinguish between cached files? if so, could you fix it, please?

Reply:
ok